contracts/upgradeable_contracts/BasicOmnibridge.sol

```diff
@@ -67,10 +66,10 @@ abstract contract BasicOmnibridge is
             name = _transformName(name);
             bridgedToken = tokenFactory().deploy(name, symbol, _decimals, bridgeContract().sourceChainId());
             _setTokenAddressPair(_token, bridgedToken);
-            _initToken(bridgedToken, _decimals);
+            _initializeTokenBridgeLimits(bridgedToken, _decimals);
```

Comment:
Will things become more complex if we would like to call relayTokensAndCall as the first bridge operation for a token? I mean, what if I would like to use relayTokensAndCall with a not-yet-bridged token?

Reply:
From my point of view, usage options of such possibility will be quite rare. However, it will increase the complexity of the code and the size of compiled contracts. Another problem might be hitting the `maxGasPerTx` limit, `deployAndHandleBridgedTokens` is already a quite expensive function (~540000 gas). Making an extra transferAndCall with callback which can consume any amount of gas can result in the undesired reverts.
What do you think? Is such possibility really required?

Reply:
Ok. I see that the increasing complexity is the most significant concern. The gas consumption of the `deployAndHandleBridgedTokens` is less significant since with the situation when users are covering gas fees for the Ethereum Mainnet we can increase the limits of the OmniBridge. Moreover, there could be a customized token implementations, so, they will consume less gas since they don't require the deployment.

But keeping the behavior consistency is another side of the сoin. From the app perspective currently there is no difference if I use relayTokens for a bridge token or for a token which is not bridged yet. It will be hard to understand why relayTokensAndCall behaves differently.